vxyowsup/whatsapp.py

```diff
@@ -65,13 +65,10 @@ def teardown_transport(self):
     def handle_outbound_message(self, message):
         # message is a vumi.message.TransportUserMessage
         log.info('Sending %r' % (message.to_json(),))
-        if message['content'] == 'fail!':
-            return self.publish_nack(message['message_id'], 'failed')
-            return self.publish_ack(
-                message['message_id'], 'remote-message-id')
-        # assumes message['to_addr'] will be the phone number
         self.stack_client.send_to_stack(
             message['content'], message['to_addr'] + '@s.whatsapp.net')
+        return self.publish_ack(
+            message['message_id'], 'remote-message-id')
```

Comment:
We might want to add a `TODO` here too to remind us to set the `remote-message-id` to something more useful (sometimes we just use `message['message_id']` again, but since the WhatsApp message has an actual id it seems polite to store that here if we can).